### PR TITLE
Update PA versions post 24.09

### PR DESCRIPTION
**Pull request by pvijayakrish**
Update version reference in PA post 24.09 release.

### CODE REVIEW

File: templates/template_vars.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 General:
-  release: 24.08
-  triton_cli_version: 0.0.11
-  genai_perf_version: 0.0.6dev
+  release: 24.09
+  triton_cli_version: 0.0.12
```
Comment (nv-hwoo) [inline templates/template_vars.yaml:3]: just to confirm, will triton cli version be bumped to 0.0.12 along with 24.09 release? (I don't see any PR or branch for the change in the triton cli repo)

Comment (pvijayakrish) [reply templates/template_vars.yaml:3]: triton_cli_version will be updated later.

Comment (nv-hwoo) [reply templates/template_vars.yaml]: should we wait for the 24.09 image so we can update both triton cli version and trtllm version at the same time?

Comment (pvijayakrish) [reply templates/template_vars.yaml]: Also this PR will be merged after 24.09 containers are published.

**Review by nv-hwoo — APPROVED** [review]
LGTM

**Pull request merged**